src/modules/cart/components/item/index.tsx

```diff
@@ -77,21 +84,16 @@ const Item = ({ item, type = "full" }: ItemProps) => {
               data-testid="product-select-button"
             >
               {/* TODO: Update this with the v2 way of managing inventory */}
-              {/* {Array.from(
+              {Array.from(
                 {
-                  length: Math.min(
-                    item.variant.inventory_quantity > 0
-                      ? item.variant.inventory_quantity
-                      : 10,
-                    10
-                  ),
+                  length: Math.min(maxQuantity, 10),
```

Comment:
Why hard-code 10?

Reply:
Just temporarily to enable changing items in cart, I'll be revamping this when I work on the inventory stuff.